src/base/dense/owl_base_dense_ndarray_generic.ml

```diff
@@ -760,7 +760,47 @@ let max' varr = (_fold_left (Pervasives.max) Pervasives.min_float varr)
 
 
 (* Sum of all elements *)
-let sum' varr = (_fold_left (+.) 0. varr)
+let sum' varr =
```

Comment:
This looks good to me. Can you also add some notes (TODO) here to remind us to further revise other functions such as min, max, and etc.